### ISSUE TITLE
Remove loss_f at Model.py

### ISSUE THREAD

**Issue by ryul99** [ryul99/pytorch-project-template#18]
Stop using the integrated `loss_f` function and use separated loss function
